common/src/main/java/com/revenuecat/purchases/common/Dispatcher.kt

```diff
@@ -38,7 +42,12 @@ open class Dispatcher(
     open fun enqueue(command: Runnable) {
         synchronized(this.executorService) {
             if (!executorService.isShutdown) {
-                executorService.execute(command)
+                if (executorService is ScheduledExecutorService) {
+                    val randomDelay = (0..JITTERING_DELAY_MILLISECONDS).random()
```

Comment:
This is adding a random delay to _all_ requests, right? 
That's not intended as it'll make the SDK effectively slower, and it could get pretty slow if you have several changed requests, which could make UIs slow. 
What we're shooting for is a random delay _only_ if requests are performed with the app in the background, but in the foreground it should remain as responsive as possible